docs/proposals/The proposal about building a high frequency Sedna based end to end use case in ModelBox format.md

```diff
@@ -0,0 +1,88 @@
+
+- **Use the expandable development function of ModelBox, use C++ to develop functional units, package Sedna Library in ModelBox, and call Sedna API to realize relevant functions.**
+- **Develop the flow chart unit on the ModelBox, call the API of the encapsulated Sedna library, and realize the incremental training helmet detection sample.**
+- **Use the ModelBox format to integrate the application case (incremental learning case).**
+
+
+
+## 4. Design Details
+
+### Flow chart of ModelBox integrated Sedna edge-cloud collaboration function
+
+![ModelBox intergrated Sedna flow chart](images\ModelBox intergrated Sedna flow chart.png)
+
+Use **ModelBox editor** visual development function to carry out unit development function. Use the **ModelBox** format to encapsulate the **k8s** service and call the interface to manage the **Sedna** edge cloud nodes. The incremental training model is realized. The data obtained by the side is trained by simple reasoning to obtain an inaccurate model. The data obtained by the side is transmitted to the cloud through **local controller**. The cloud performs incremental training through **global manager** to obtain a more accurate model through reasoning. The inference results of the side cloud model are output through **httpserver**. **K8s** library is encapsulated by ModelBox, and Sedna functions such as edge reasoning training and cloud incremental training are encapsulated by function unit development function.
+
+### 1. Specific development process of functional unit of ModelBox
+
+#### 1. Function unit creation
+
+ The directory structure of the created **C++** functional unit is as follows:
+
+```
+[flowunit-name]
+     |---CMakeList.txt           # Compile files,and C++ functional units are compiled with cmake
+     |---[flowunit-name].cc      # Interface implementation file
+     |---[flowunit-name].h       # Header file
+     |---[flowunit-name].toml    # Configuration file for webui display
+```
+
+#### 2. Function unit attribute configuration
+
+#### 3. Logical realization of functional unit
+
+#### 4. Function unit compilation and operation
+
+The ModelBox framework C++ project is compiled using **cmake**. The functional units created through the command line or visual UI contain`cmakelists.txt` file by default. The main functions are as follows:
+
+- Set function unit name
+- Header files required for linking functional units
+- Libraries required for linking functional units
+- Set compilation target to dynamic library
+- Specify the function unit installation directory
+
+**View [official website document for details]( https://modelbox-ai.com/ )**
+
+
+
+### 2. Application case of encapsulated Sedna
+
+#### 1. Create incremental learning
+
+#### 2. Start incremental training
+
+**View [use incremental learning in helmet detection scenario for specific details](https://github.com/kubeedge/sedna/blob/main/examples/incremental_learning/helmet_detection/README.md)**
+
+
+
+## 5.Road Map
```

Comment:
Road Map should not be included here.

Reply:
It is fine to have a road map though

Reply:
ok, it can be used for tracking the process of development.